### PR TITLE
xdiskusage: init at 1.60

### PR DESCRIPTION
**Pull request by fuzzdk**
## Description of changes

New package: xdiskusage. Simple program to view disk usage or other data formatted as the output of du.
https://xdiskusage.sourceforge.net/

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/xdiskusage/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchurl, fltk }:
+
+stdenv.mkDerivation rec {
+  pname = "xdiskusage";
+  version = "1.60";
+
+  src = fetchurl {
+    urls = [
```
Comment (AndersonTorres) [inline pkgs/tools/misc/xdiskusage/default.nix:8]: If there is only one url, use `url`.

File: pkgs/tools/misc/xdiskusage/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "https://xdiskusage.sourceforge.net/${pname}-${version}.tgz"
```
Comment (AndersonTorres) [inline pkgs/tools/misc/xdiskusage/default.nix:9]: Do not parameterize `pname`:

```suggestion
      "https://xdiskusage.sourceforge.net/xdiskusag -${version}.tgz"
```

File: pkgs/tools/misc/xdiskusage/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    ];
+    hash = "sha256-e1NtxvG9xtm+x8KblDXCPZ0yv/ig6+4muZZrJz3J9n4=";
+  };
+
+  nativeBuildInputs = [ fltk ];
```
Comment (AndersonTorres) [inline pkgs/tools/misc/xdiskusage/default.nix:14]: Afaicr, it should be a `buildInput`.

File: pkgs/tools/misc/xdiskusage/default.nix [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchurl, fltk }:
+
+stdenv.mkDerivation rec {
```
Comment (AndersonTorres) [inline pkgs/tools/misc/xdiskusage/default.nix:3]: Have you tried

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix [hunk 5]
```diff
@@ -5853,6 +5853,11 @@
     githubId = 17859309;
     name = "Fuzen";
   };
+  fuzzdk = {
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:5856]: Put this in a separate commit (no need for a new PR).

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/xdiskusage/default.nix (hunk 3 above)
Comment (fuzzdk) [reply pkgs/tools/misc/xdiskusage/default.nix:14]: I think (but may be wrong) it should be nativeBuildInput as it provides the bin/fluid that is used at build time.

File: maintainers/maintainer-list.nix (hunk 5 above)
Comment (fuzzdk) [reply maintainers/maintainer-list.nix:5856]: done

**Review by AndersonTorres — APPROVED** [review]
Ping me 36h to merge.

**Comment by fuzzdk** [comment]
> Ping me 36h to merge.

ping @AndersonTorres
